package.json

```
@@ -0,0 +1,54 @@
+{
+  "name": "@evolv/javascript-sdk",
+  "version": "1.0.0",
+  "description": "This is the official JavaScript client for the Evolv Technologies optimization platform.",
+  "main": "dist/index.js",
+  "module": "src/index.js",
+  "type": "module",
+  "homepage": "https://www.evolv.ai",
+  "bugs": {
+    "url": "https://github.com/evolv-ai/javascript-sdk/issues",
+    "email": "support@evolv.ai"
+  },
+  "directories": {
+    "doc": "./docs"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/evolv-ai/javascript-sdk"
+  },
+  "scripts": {
+    "test": "mocha",
+    "build": "webpack --config=webpack.dev.cjs",
+    "build-prod": "webpack --config=webpack.prod.cjs --display-modules",
+    "docs": "jsdoc -c jsdocs.config.json src/index.js src/docs/README.md"
+  },
+  "author": {
+    "name": "Evolv Technologies",
+    "email": "support@evolv.ai"
+  },
+  "license": "MIT",
+  "dependencies": {
+    "base64-arraybuffer": "^0.2.0",
+    "deepmerge": "^4.2.2"
+  },
+  "devDependencies": {
+    "chai": "^4.2.0",
+    "chai-spies": "^1.0.0",
+    "jsdoc": "^3.6.3",
+    "mocha": "^7.1.0",
+    "webcrypto": "git+https://github.com/nodejs/webcrypto.git",
+    "webpack": "^4.42.0",
+    "webpack-cli": "^3.3.11",
+    "webpack-merge": "^4.2.2",
+    "xhr-mock": "^2.5.1",
+    "xmlhttprequest": "^1.8.0"
+  },
+  "mocha": {
+    "extension": [
+      "js"
+    ],
+    "spec": "src/tests/*.test.js",
```

You have tests in _src/ponyfills/tests_ as well.
```suggestion
    "spec": "src/**/*.test.js",
```